cypress/samples/timings.csv

```diff
@@ -0,0 +1,3 @@
+Title,Episodes Pubdate,The Guid,And The Midroll Breaks
+Title,2023-07-05,prx_284_81888abe-4ae1-4708-ad50-7f99149cd710,"{8,12}"
+S2 EP2 Glassware,2021-03-03,prx_284_b1a9ec50-cd9e-4dfd-8a63-1fe1986db1cb,{5}
```

Comment:
@radical-ube It looks like this file doesn't have a [newline at the end](https://github.com/PRX/acceptance-tests/blob/main/.editorconfig#L7). This isn't strictly a problem, but it does likely indicate that your code editor or IDE isn't observing the `.editorconfig` file in the project. Following the rules in .editorconfig for any project that has one is [a good practice](https://github.com/PRX/internal/wiki/Guide:-Local-Development-Environment#set-up-ide), since it will ensure some global style consistencies that aren't otherwise covered by more specific formatters or linters that we use. I would recommend configuring your editor to follow .editorconfig rules any time it's present.